script_test.go

```diff
@@ -2284,6 +2382,20 @@ func ExampleSlice() {
 	// 3
 }
 
+func ExamplePipe_EncodeBase64() {
+	script.Echo("hello\nthere\nworld\n").EncodeBase64().Stdout()
+	// Output:
+	// aGVsbG8KdGhlcmUKd29ybGQK
+}
+
+func ExamplePipe_DecodeBase64() {
+	script.Echo("aGVsbG8KdGhlcmUKd29ybGQK").DecodeBase64().Stdout()
```

Comment:
Just out of curiosity, how do we handle base64 input that's padded with one or two `=` characters?  `StdEncoding` does use padding, it seems, but we don't have any examples in the test cases. It would be weird if it didn't work, since we're using the `stdlib` for the implementation, of course, but that's the point of tests: they should specify behaviour without relying on any assumptions about implementation details.

Reply:
@bitfield the `single line string` test case does have a decoded input with padding (`aGVsbG8gd29ybGQ=`). does this work?